source/npm/util.js

```diff
@@ -164,6 +164,10 @@ async function getFilesIgnoredByDotnpmignore(pkg, fileList) {
 
 function filterFileList(globArray, fileList) {
 	const globString = globArray.length > 1 ? `{${globArray}}` : globArray[0];
+	if (!globString) {
+		return [];
+	}
+
```

Comment:
Shouldn't we do this more explicitly? i.e.

```suggestion
	if (globArray.length === 0) {
		return [];
	}

	const globString = globArray.length > 1 ? `{${globArray}}` : globArray[0];
```